### DISCUSSION TITLE
x-if x-elseif x-else

### DISCUSSION THREAD

**Discussion by youfeed** [alpinejs/alpine#1759] [7. Bugs (archived)]
### Discussed in https://github.com/alpinejs/alpine/discussions/1743

<div type='discussions-op-text'>

<sup>Originally posted by **youfeed** July  8, 2021</sup>
x-if x-elseif x-else</div>

**Comment by pohlt** [comment]
Actually, this is not a bug but a feature request at best. :-)

I would need something like `x-else` as well, but wouldn't know how this could work, TBH.
